### DISCUSSION TITLE
RAG document read error

### DISCUSSION THREAD

**Discussion by flyfox666** [open-webui/open-webui#3843] [General]
After upgrading to 0.3.7 and 0.3.8, there is a strange problem with the Q&amp;A of RAG documents.

The first conversation after uploading a document reads the document and can be answered correctly, but a subsequent question cannot be linked to the document.

Downgrading from a 0.3.8 document to 0.3.7 doesn't work either, while the log display issue in the current 0.3.8 is not yet fixed in the stable release

![Screenshot 2024-07-13 151243](https://github.com/user-attachments/assets/cc9f9764-901d-4303-b090-c613016605fa)

**Comment by flyfox666** [comment]
Is this a problem caused by ChromaDB?

**Comment by ProjectMoon** [comment]
It is an intentional change from 0.3.6: https://github.com/open-webui/open-webui/discussions/3516#discussioncomment-10024991

Related: in 0.3.8, I've also noticed that models don't seem to use document contexts on first responses. Subsequent ones work fine, though o_O

**Reply by flyfox666** [reply]
Thanks a lot , really strange .let's just waitting for the 3.10
